### ISSUE TITLE
Check if file input were empty before starting upload

### ISSUE THREAD

**Issue by JonGarbayo** [balderdashy/skipper#92]
Hi,

I'm using Sails 11.1 and skipper 2.11.2.
I have a form to create an article in my app. An image can be attached to an article, so there is an input file.

I would like to check, server-side, if this input file was empty, before trying to start upload.

Is it possible, and if yes, how can I do this?

I tried to check `req.files`, or `req.files.photo` ("photo" is the input name) and `req.body.photo`, but they are always undefined, even if there is a file.
`req.file('photo')` is always defined, so I can't check it. 

Just a note: when the input is filled with a file, the upload works.

Thanks in advance.

**Comment by JonGarbayo** [comment]
For those who have the same issue, check the answer [here](http://stackoverflow.com/questions/31061719/sails-js-with-skipper-check-if-file-input-were-empty-before-starting-upload).

There is now way to do differently for the moment.

**Comment by sailsbot** [comment]
Thanks for posting, @JonGarbayo.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
